curious/api.py

```diff
@@ -18,9 +17,9 @@
 CACHE_VERSION = 5
 CACHE_TIMEOUT = 60*60
 try:
-  cache = get_cache('curious')
+  cache = caches['curious']
 except:
-  cache = get_cache('default')
+  cache = caches['default']
```

Comment:
This was just a flat port of the existing code. It's likely a KeyError, but I opted not to go with the `.get` because caches is likely a proxy object and just looks like a dict.

Reply:
Yup:
```
>>> caches.get('curious')
Traceback (most recent call last):
  File "<console>", line 1, in <module>
AttributeError: 'CacheHandler' object has no attribute 'get'
```
I used the actual error code though (`InvalidCacheBackendError`)